posts/2022-08-19-outreachy-wrap-up-blog-2022.rst

```diff
@@ -0,0 +1,68 @@
+.. post:: 2022-08-19
+  :tags: Outreachy
+  :author: Surbhi Sharma
+  :redirect: 2022/08/19/outreachy-wrap-up-blog-2022
```

Comment:
```suggestion
.. post:: 2022-08-26
  :tags: Outreachy
  :author: Surbhi Sharma
  :redirect: 2022/08/26/outreachy-wrap-up-blog-2022
```

Let's fix the date (assuming we merge tomorrow) on this metadata block _and_ the filename.